.circleci/config.yml

```diff
@@ -33,8 +33,19 @@ aliases:
 
 version: 2
 jobs:
-
+  generate-spec:
```

Comment:
This job generates from schema and not the spec, so should probably rename it.

Reply:
👍